Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -31,8 +31,20 @@ struct PlaceDataSourceError: Error {
 
 class PlaceCarouselViewController: UIViewController {
 
+    fileprivate let minimumTimeAtLocationBeforeFetchingEvents: TimeInterval = 15 * 60
+    fileprivate let currentLocationIdentifier = "CURRENT_LOCATION"
     fileprivate let MIN_SECS_BETWEEN_LOCATION_UPDATES: TimeInterval = 1
-    fileprivate var timeOfLastLocationUpdate: Date?
+
+    private let timeOfLastLocationUpdateKey = "timeOfLastLocationUpdate"
+
+    fileprivate var timeOfLastLocationUpdate: Date? {
+        get {
+            return UserDefaults.standard.value(forKey: timeOfLastLocationUpdateKey) as? Date
```

Comment:
On a fresh install, I think `timeOfLastLocationUpdate` will always be nil.

Also, I can't seem to find the `UserDefaults.value` method – the one I found was [`object`](https://developer.apple.com/reference/foundation/userdefaults/1410095-object). Know what might be going on there?